pkg/properties/propertiesSet_test.go

```diff
@@ -169,7 +169,7 @@ func TestUpdatePropertyWithValidNamespaceAndVaidNameValueReturnsOk(t *testing.T)
 
 func TestUpdatePropertyWithInvalidNamesapceAndValidNameReturnsError(t *testing.T) {
 	//Given...
-	namespace := "invalidNamespace"
+	namespace := "invalidnamespace"
```

Comment:
why would this be invalid?

Reply:
I see the issue, I have changed it to have a special character